index.bs

```diff
@@ -264,7 +264,7 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl
     type: dfn;
         text: get a copy of the bytes held by the buffer source; url: dfn-get-buffer-source-reference
 
-spec: webdriver; urlPrefix: https://w3c.github.io/webdriver/webdriver-spec.html#
+spec: webdriver; urlPrefix: https://w3c.github.io/webdriver/#
```

Comment:
```suggestion
spec: webdriver; urlPrefix: https://w3c.github.io/webdriver
```

Reply:
You're right that `#` isn't needed, but dropping the slash will lead to an extra redirect. I'll apply half of this suggestion :)